ref/general/oidc.adoc

```diff
@@ -0,0 +1,77 @@
+:page-layout: general-reference
+:page-type: general
+:page-description: OpenID Connect (OIDC) is an identity protocol based on OAuth 2.0 that is widely used in SSO technologies. OIDC works well with cloud, mobile, and native applications. 
+:page-categories: OpenID Connect
+:seo-title: OpenID Connect (OIDC) is an identity protocol based on OAuth 2.0.
+:seo-description: OpenID Connect (OIDC) is an identity protocol based on OAuth 2.0 that is widely used in SSO technologies. OIDC works well with cloud, mobile, and native applications.
+
+= Configuring Open Liberty as an SSO provider in a dev/test environment
+
+In an increasingly fragile world of the internet, user credentials are precious resources that need to be protected. With single sign-on (SSO), users can log in easily without worrying that that their credentials are being exposed. Open Liberty helps you with identity management.
+
+You can configure Open Liberty as an SSO provider, so that web users can authenticate once when accessing Liberty resources such as HTML, JavaServer Pages (JSP) files, and servlets, or accessing resources in multiple Liberty servers that share the same Lightweight Third Party Authentication (LTPA) keys.
```

Comment:
this sentence should be broken up into 2 or 3 shorter sentences- too long for acrolinx at 46 words